catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/common/CsexraAttributeTest.java

```diff
@@ -229,12 +229,873 @@ public void testSnowDepthMaxCategoryOther() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.A_S_VERT_GSD_NAME)).thenThrow(NitfFormatException.class);
+
+        Serializable actual = CsexraAttribute.A_S_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+    }
+
+    @Test
+    public void testCSVertGSDMin() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.C_S_VERT_GSD_NAME)).thenReturn("000.0");
+
+        Serializable actual = CsexraAttribute.C_S_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("000.0"));
+
+    }
+
+    @Test
+    public void testCSVertGSDMax() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.C_S_VERT_GSD_NAME)).thenReturn("999.9");
+
+        Serializable actual = CsexraAttribute.C_S_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("999.9"));
+    }
+
+    @Test
+    public void testCSVertGSDNA() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.C_S_VERT_GSD_NAME)).thenReturn("N/A");
+
+        Serializable actual = CsexraAttribute.C_S_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("N/A"));
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testCSVertGSDNotSet() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.C_S_VERT_GSD_NAME)).thenThrow(NitfFormatException.class);
+
+        Serializable actual = CsexraAttribute.C_S_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+    }
+
+    @Test
+    public void testGeoMeanVertGSDMin() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.GEO_MEAN_VERT_GSD_NAME)).thenReturn("000.0");
+
+        Serializable actual = CsexraAttribute.GEO_MEAN_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("000.0"));
+
+    }
+
+    @Test
+    public void testGeoMeanVertGSDMax() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.GEO_MEAN_VERT_GSD_NAME)).thenReturn("999.9");
+
+        Serializable actual = CsexraAttribute.GEO_MEAN_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
+        assertThat(actual, is("999.9"));
+    }
+
+    @Test
+    public void testGeoMeanVertGSDNA() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.GEO_MEAN_VERT_GSD_NAME)).thenReturn("N/A");
+
+        Serializable actual = CsexraAttribute.GEO_MEAN_VERT_GSD.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, is(instanceOf(String.class)));
```

Comment:
Should I remove all of the instanceOf tests? Those were part of the test class, so I kept with the existing pattern.